modules/php/Cards/Rules/RuleCardFactory.php

```diff
@@ -2,21 +2,35 @@
 
 namespace Fluxx\Cards\Rules;
 use Fluxx\Cards\CardFactory;
+use Fluxx\Game\Utils;
 /*
  * RuleCardFactory: how to create Rule Cards
  */
 class RuleCardFactory extends CardFactory
 {
   public static function getCardFullClassName($uniqueId)
   {
-    $name = "Fluxx\Cards\Rules\\" . self::$classes[$uniqueId];
+    if (array_key_exists($uniqueId, self::$classesCreeperPack))
```

Comment:
Rather than doing that, we could also simply add a parameter in the Goal, Action and Rule card like `creeperPack`, defaulting to false. And when you import, ignore/include the creeperPack depending on the option?